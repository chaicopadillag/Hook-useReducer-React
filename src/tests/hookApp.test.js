import React from 'react';
import { shallow } from 'enzyme';
import HookApp from '../HookApp';

describe('Prueba de HooApp', () => {
	test('Deber de hacer un To Match Snapshot', () => {
		const wraper = shallow(<HookApp />);
		expect(wraper).toMatchSnapshot();
	});
});
