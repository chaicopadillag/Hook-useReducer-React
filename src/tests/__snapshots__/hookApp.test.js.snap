// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de HooApp Deber de hacer un To Match Snapshot 1`] = `
<div>
  <h1>
    Hook App
  </h1>
</div>
`;
